bundledata.go

```diff
@@ -234,6 +234,19 @@ type ApplicationSpec struct {
 
 	// EndpointBindings maps how endpoints are bound to spaces
 	EndpointBindings map[string]string `bson:"bindings,omitempty" json:"bindings,omitempty" yaml:"bindings,omitempty"`
+
+	// Plans specifies plans under which the application is to be deployed.
+	Plans PlansSpec `bson:"plans,omitempty" json:"plans,omitempty" yaml:"plans,omitempty"`
+}
+
+// PlansSpec represents plans under which an application will be deployed.
+type PlansSpec struct {
+	// Plan specifies a plan url under which the application will be
+	// deployed.
+	Plan string `bson:"plan,omitempty" json:"plan,omitempty" yaml:"plan,omitempty"`
+	// UseDefault is true if the application is to be deployed under
+	// its default plan.
+	UseDefault bool `bson:"use-default,omitempty" json:"use-default,omitempty" yaml:"use-default,omitempty"`
```

Comment:
If true, the plan name must be empty-string, or validation should fail.

Reply:
:bike: :bike: :bike: :bike: :bike: :bike: :bike: :bike: :bike: :bike: :bike: :bike: :bike: :bike: :bike: 
You could just specify a single field 
```
Plan string
```
And have empty string mean "use default plan"